assets/thanos-ruler/kube-rbac-proxy-secret.yaml

```diff
@@ -1,10 +1,19 @@
 apiVersion: v1
-data:
-  config.yaml: ImF1dGhvcml6YXRpb24iOgogICJyZXNvdXJjZUF0dHJpYnV0ZXMiOgogICAgImFwaVZlcnNpb24iOiAibWV0cmljcy5rOHMuaW8vdjFiZXRhMSIKICAgICJuYW1lc3BhY2UiOiAie3sgLlZhbHVlIH19IgogICAgInJlc291cmNlIjogInBvZHMiCiAgInJld3JpdGVzIjoKICAgICJieVF1ZXJ5UGFyYW1ldGVyIjoKICAgICAgIm5hbWUiOiAibmFtZXNwYWNlIg==
+data: {}
 kind: Secret
 metadata:
   labels:
-    k8s-app: thanos
+    k8s-app: thanos-ruler
   name: thanos-ruler-kube-rbac-proxy
   namespace: openshift-user-workload-monitoring
+stringData:
+  config.yaml: |-
+    "authorization":
+      "resourceAttributes":
+        "apiVersion": "monitoring.coreos.com/v1"
+        "namespace": "{{ .Value }}"
+        "resource": "prometheusrules"
+      "rewrites":
+        "byQueryParameter":
+          "name": "namespace"
```

Comment:
very good idea :+1: now this is readable.

Reply:
stolen from @brancz ;)

https://github.com/coreos/kube-prometheus/commit/23344a39eb3f7a598d395e9185ca597cd4a60b1d#diff-80dac8ff1487b9e85adef38dd4356cb7R68-R69